.github/workflows/pull-request.yml

```diff
@@ -243,17 +247,20 @@ jobs:
       - uses: ./.github/actions/short-sha
 
       - name: Terraform destroy
+        env:
+          TARGET_ENV: ${{ needs.build_base.outputs.target_env || 'test' }}
         run: |
+          [[ -f uhd.sh ]] || { echo "uhd.sh not found! Exiting."; exit 1; }
           source uhd.sh
           uhd terraform init:layer 20-app
-          uhd terraform destroy:layer 20-app ci-$SHORT_SHA
+          uhd terraform destroy:layer 20-app "$TARGET_ENV-ci-$SHORT_SHA"
```

Comment:
All our ci environments are labelled as `ci-<short commit hash>`
This is going to run the destroy command against a workspace that doesn't exist

Reply:
Agreed and reverted to previous format